streams/examples/src/main/java/org/apache/kafka/streams/examples/pageview/PageViewTypedDemo.java

```diff
@@ -100,6 +101,8 @@ public static void main(String[] args) throws Exception {
         serdeProps.put("JsonPOJOClass", PageView.class);
         pageViewDeserializer.configure(serdeProps, false);
 
+        final Serde<PageView> pageViewSerde = Serdes.serdeFrom(pageViewSerializer, pageViewDeserializer);
```

Comment:
Looks good.  Much better to construct the full serde up front then to squeeze it into the `stream` or `table` calls later.